Makefile

```diff
@@ -92,3 +92,23 @@ oam-runtime.help:
 help-special: oam-runtime.help
 
 .PHONY: oam-runtime.help help-special
+
+generate: controller-gen
+	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
```

Comment:
I don't think we need this - the our build submodule make lib calls `go generate`, which calls this per https://github.com/crossplane/oam-kubernetes-runtime/blob/2cccfe365030ec41e27d78ef356f0fc9281816f3/apis/generate.go#L25

Reply:
how does this work? I think that the goal is to make this repro independent enough that anyone who is interested in OAM can come to contribute. I am trying to set up a generic system to achieve that and this is just the first step.